src/components/Notifications/Notification.js

```diff
@@ -44,19 +39,19 @@ export const Notification = (props) => {
     );
   };
 
-  const wrapToastContent = content =>
+  const wrapToastContent = content => (
     <div className="slds-notify__content">
       {content}
-    </div>;
+    </div>
+  );
 
   return (
     <div className="slds-notify_container">
       <div {...rest} className={cx(sldsClasses)} role="alert">
         {icon && wrapIcon(icon)}
         <Button
-          icon-inverse={getThemeName(className) ? undefined : true}
+          flavor={[(theme === 'warning') ? null : 'icon-inverse', 'icon']}
```

Comment:
See above

Reply:
&& returns `falsy`, but I want `null`.

Reply:
no, you push a needless null key into the array here